tests/unit/store/entity/actions.spec.ts

```diff
@@ -49,4 +51,18 @@ describe( 'entity/actions', () => {
 		} );
 	} );
 
+	it( DETERMINE_EDITABILITY, () => {
+		factory.setEntityEditabilityResolver( {
+			isEditable: () => Promise.resolve( true ),
+		} );
+
+		const context = {
+			commit: jest.fn(),
+		};
+
+		return ( actions[ DETERMINE_EDITABILITY ] as any )( context ).then( () => {
+			expect( context.commit ).toHaveBeenCalledWith( EDITABILITY_UPDATE, true );
```

Comment:
The pickiest of nitpicks: the `true` in a variable would make the test even more self-explanatory.

Reply:
done